packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -786,11 +950,9 @@ const LoadTestDataStoreInstantiationFactory = new DataObjectFactory(
 export const createFluidExport = (runtimeOptions: IContainerRuntimeOptions) =>
 	new ContainerRuntimeFactoryWithDefaultDataStore({
 		defaultFactory: LoadTestDataStoreInstantiationFactory,
-		registryEntries: new Map([
-			[
-				LoadTestDataStore.DataStoreName,
-				Promise.resolve(LoadTestDataStoreInstantiationFactory),
-			],
-		]),
+		registryEntries: [
```

Comment:
Can we use both Map and array here?

Reply:
I'm using `PureDataObjectFactory.registryEntry` which does the same exact thing.